analysis/test/src/pointers.rs

```diff
@@ -508,6 +516,7 @@ unsafe fn main_0(mut argc: libc::c_int, mut argv: *mut *mut libc::c_char) -> lib
     analysis2();
     inter_function_analysis();
     no_owner(0i32);
+    free(global as *mut libc::c_void);
```

Comment:
The leak in `no_owner` when called with `no_owner(0)` is intentional.  Should it be freed in `main_0`, or is it intentional to completely leak it?

Reply:
i think what you have here is fine